src/app.js

```diff
@@ -0,0 +1,46 @@
+import "regenerator-runtime/runtime";
+import axios from "axios";
+
+const BASE = "https://api.github.com/orgs/stackbuilders/repos";
+
+const getAllItems = async () => {
+  try {
+    const response = await axios.get(BASE);
+
+    const allItems = response.data;
```

Comment:
Lets parse this huge object into something more manageable, we can get rid of most of the keys the object has and only take those we care about.

Reply:
I tried a new function "someItems" to get part of the array but it did not work at all, on Monday I will continue fixing it to gain the objective.

Reply:
It is already implemented the function in the correct way, can you check it?